config/initializers/kaui.rb

```diff
@@ -44,6 +44,7 @@ module Kaui
 Kaui.plugins_whitelist ||= ENV.fetch('KAUI_PLUGINS_WHITELIST', nil)
 Kaui.root_username ||= (ENV['KAUI_ROOT_USERNAME'] || 'admin')
 Kaui.disable_sign_up_link ||= (ENV['KAUI_DISABLE_SIGN_UP_LINK'] || 'true') == 'true'
+Kaui.additional_headers_partial = 'layouts/additional_header'
```

Comment:
Does it need to be `layouts/_additional_header`? 🤔

Reply:
the partial render will add _ to the file name, so it should be layouts/additional_header  :smile: